web/.npmrc

```diff
@@ -1 +1 @@
-engine-strict=true
+engine-strict=true
```

Comment:
is it the good formating or an artefact from your code editor ?
(same with web/.nvmrc but for some reason i cannot add a comment there)